thefuck/rules/git_clone_missing.py

```diff
@@ -0,0 +1,42 @@
+'''
+Rule: git_clone_missing
+
+Correct missing `git clone` command when pasting a git URL
+
+```sh
+& https://github.com/nvbn/thefuck.git
+git clone https://github.com/nvbn/thefuck.git
+
+Author: Miguel Guthridge
+'''
+from urllib import parse
```

Comment:
We still support py2:

```suggestion
from six.moves.urllib import parse
```